config.js

```diff
@@ -238,114 +238,130 @@ exports.updateFromArgv = function() {
-    }
-
-    if (argv.maxmaintime || argv.maxmaintimeranked || argv.maxmaintimeunranked || argv.minmaintime || argv.minmaintimeranked || argv.minmaintimeunranked) {
-        console.log("Warning: --min/max*maintime*+/-ranked/unranked is not supported anymore\n Use --min/max*maintime*blitz/live/corr*+/-ranked/unranked");
-    }
-
-    if (argv.maxperiodtime || argv.maxperiodtimeranked || argv.maxperiodtimeunranked || argv.minperiodtime || argv.minperiodtimeranked || argv.minperiodtimeunranked) {
-        console.log("Warning: --min/max*periodtime*+/-ranked/unranked is not supported anymore\n Use --min/max*periodtime*blitz/live/corr*+/-ranked/unranked");
-    }
-
-    if (argv.maxperiods && (argv.maxperiodsranked || argv.maxperiodsunranked)) {
-        console.log("Warning: You are using --maxperiods in combination with --maxperiodsranked and/or --maxperiodsunranked.\nUse either --maxperiods alone, OR --maxperiodsranked with --maxperiodsunranked.\nBut don't use the 3 maxperiods arguments at the same time.");
-    }
-
-    if (argv.minperiods && (argv.minperiodsranked || argv.minperiodsunranked)) {
-        console.log("Warning: You are using --minperiods in combination with --minperiodsranked and/or --minperiodsunranked.\nUse either --minperiods alone, OR --minperiodsranked with --minperiodsunranked.\nBut don't use the 3 minperiods arguments at the same time.");
-    }
+    /*** console : DEPRECIATIONS ***/
+    console.log("-----DEPRECIATIONS-----");
+    this.indicatorConsoleDepreciation = false;
 
-    if (argv.minrank && (argv.minrankranked || argv.minrankunranked)) {
-        console.log("Warning: You are using --minrank in combination with --minrankranked and/or --minrankunranked. \n Use either --minrank alone, OR --minrankranked with --minrankunranked.\nBut don't use the 3 minrank arguments at the same time.");
-    }
-
-    if (argv.maxrank && (argv.maxrankranked || argv.maxrankunranked)) {
-        console.log("Warning: You are using --maxrank in combination with --maxrankranked and/or --maxrankunranked. \n Use either --maxrank alone, OR --maxrankranked with --maxrankunranked.\nBut don't use the 3 maxrank arguments at the same time.");
-    }
-
-    if (argv.nopause && (argv.nopauseranked || argv.nopauseunranked)) {
-        console.log("Warning: You are using --nopause in combination with --nopauseranked and/or --nopauseunranked. \n Use either --nopause alone, OR --nopauseranked with --nopauseunranked.\nBut don't use the 3 nopause arguments at the same time.");
-    }
-
-    if (argv.komi) {
-        if (argv.komi.includes(`auto`)) {
-            console.log("Warning: /--komi auto/ has been renamed to /--komi automatic/\n");
-            // we skip a line here, not below, because this argv may be undefined if not used by bot admin
+    function depreciatedArgumentIsUsedNewArgumentToUseInstead(depreciatedArgumentUsed, newArgumentToUseInstead, indicatorDepreciation) {
```

Comment:
Also, since this function is only used locally, we don't have to give it quite as long a name - in my opinion.

Generally with variable naming - the things you use only on a few nearby lines can have a single-character name, things you use in a function an abbreviation. Functions, object attributes and other things used across a whole module/file should probably have clear names and global cross-module names (or APIs even) should be extra careful to be clear.

But do always ask if a part of the name can be removed without losing clarity. I think in this case for example `depreciatedArgumentWarning()` is sufficient and since it is only used in this function, you might be able to make it shorter still without causing any confusion.